api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -40,6 +41,8 @@
 
   private static final String WORKSPACE_CDR_ENV_KEY = "WORKSPACE_CDR";
   private static final String JUPYTER_DEBUG_LOGGING_ENV_KEY = "JUPYTER_DEBUG_LOGGING";
+  private static final String MERGED_WGS_BUCKET_KEY = "MERGED_WGS_BUCKET";
```

Comment:
feel free to nitpick names

Reply:
should we just directly refer to the file here instead of the bucket? 

Also, how about "ALL_SAMPLES" over "MERGED"?

Reply:
`ALL_SAMPLES` works.

I wanted to refer to the bucket because for single sample files it's a bucket with all the files and I wanted it to be consistent. That's probably not a good enough reason for it to not be the file for the all-samples file though.

Reply:
oh, actual answer as to why bucket: it's sharded